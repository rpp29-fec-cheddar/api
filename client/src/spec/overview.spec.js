/*eslint-env es6*/
/* eslint-disable no-unused-vars */
/* eslint-disable-next-line func-style */
/*eslint func-style: ["error", "declaration"]*/
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../app.jsx';
import Overview from '../components/overview/overview.jsx';


// eslint-disable-next-line no-undef
describe('My Test Suite', () => {
  // eslint-disable-next-line no-undef
  test('My Test Case', () => {
    // eslint-disable-next-line no-undef
    expect(true).toEqual(true);
  });
});

// eslint-disable-next-line func-style
function sum(a, b) {
  return a + b;
}

// eslint-disable-next-line no-undef
test('adds 1 + 2 to equal 3', () => {
  // eslint-disable-next-line no-undef
  expect(sum(1, 2)).toBe(3);
});


// eslint-disable-next-line no-undef
describe('App', () => {
  test('Test that App renders', () => {
    const component = renderer.create(<App ></App>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Overview', () => {
  test('Test that Overview renders', () => {
    const component = renderer.create(<Overview ></Overview>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});