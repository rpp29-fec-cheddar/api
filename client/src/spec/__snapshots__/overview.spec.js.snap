// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test that App renders 1`] = `
<div>
  <h1
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    H1 Element
  </h1>
  <br />
  <br />
  <br />
  <br />
  <br />
  <br />
</div>
`;

exports[`Overview Test that Overview renders 1`] = `
<div>
  <div>
    Overview Here!
  </div>
  <div />
  <div />
  <div />
  <div />
  <div />
  <br />
  <br />
  <br />
  <div />
</div>
`;
